VersionMiner/ActionInputs.cs

```diff
@@ -27,6 +27,16 @@ public class ActionInputs
         HelpText = "Gets or sets the name of the repository.")]
     public string RepoName { get; set; } = string.Empty;
 
+    /// <summary>
+    /// Gets or sets the token used to authenticate to the repository owner set by the <see cref="RepoOwner"/> value
+    /// for a repository name set by the <see cref="RepoName"/> value.
```

Comment:
Refactor code docs to below:

```suggestion
    /// Gets or sets the token used to authenticate to the repository set by <see cref="RepoName"/> which is owned by the <see cref="RepoOwner"/>.
```